cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -233,6 +236,13 @@ renderShelleyTxCmdError err =
     ShelleyTxCmdByronEraQuery -> "Query not available in Byron era"
     ShelleyTxCmdLocalStateQueryError err' -> renderLocalStateQueryError err'
     ShelleyTxCmdBalanceTxBody err' -> Text.pack $ displayError err'
+    ShelleyTxCmdExpectedKeyLockedTxIn txins ->
+      "Expected key witnessed tx inputs but got script witnessed tx inputs: " <>
+      Text.singleton '\n' <>
+      Text.intercalate (Text.singleton '\n') (map renderTxIn txins)
+    ShelleyTxCmdTxInsDoNotExist txins ->
+      "The following tx input(s) were not present in the UTxO: " <>
```

Comment:
Imo it's easier to read if each missing inout is on a new line (not just from the 2nd one), so add new line at end of message.

Reply:
I thought I fixed this. Good catch I'll re-push and bors.